File: src/main/java/com/vk/lingvobot/states/SettingsSetupState.java
package com.vk.lingvobot.states;

import com.vk.api.sdk.client.actors.GroupActor;
import com.vk.lingvobot.entities.DialogState;
import com.vk.lingvobot.entities.User;
import com.vk.lingvobot.entities.UserDialog;

public interface SettingsSetupState {
    void handle(User user, GroupActor groupActor, DialogState dialogState, UserDialog greetingSetUpDialog);
}

File: src/main/kotlin/com/vk/lingvobot/states/SettingsSetupState.kt
package com.vk.lingvobot.states

import com.vk.api.sdk.client.actors.GroupActor
import com.vk.lingvobot.entities.DialogState
import com.vk.lingvobot.entities.User
import com.vk.lingvobot.entities.UserDialog

interface SettingsSetupState {
    fun handle(
        user: User,
        groupActor: GroupActor,
        dialogState: DialogState,
        greetingSetUpDialog: UserDialog
    )
}